doc/tutorial/get_started.rst

```diff
@@ -131,7 +131,7 @@ Complete the following steps to create the required disks in a LXD storage pool:
 3. Create a network
 -------------------
 
-MicroCloud requires an uplink network that the cluster members can use to communicate.
+MicroCloud requires an uplink network that the cluster members can use for outside connectivity.
```

Comment:
We use the term "external" in the microcloud init process, shall we use the same term here too?